tests/test_upgrade_shapella.py

```diff
@@ -328,9 +331,18 @@ def test_vote(
         events_update_oracle_impl,
         events_grant_staking_router_role,
         events_template_finish,
-        *revoke_roles_events,
+        *remaining_events,
```

Comment:
maybe it's better to have two groups of events to avoid magic-border slices below?

Reply:
How do you see it? I don't catch.
And "magical" slices is not scary at all here. It's properly checked afterwards that we sliced exactly what is meant  to be sliced.

Reply:
It's not so important to me, so I took your point :+1: